client/asset/dcr/dcr.go

```diff
@@ -2048,6 +2084,33 @@ func (dcr *ExchangeWallet) fatalFindRedemptionsError(err error, contractOutpoint
 	dcr.findRedemptionMtx.Unlock()
 }
 
+// blockMaybeContainsScripts uses the cfilters of the specified block to
+// determine if the block likely includes any of the passed scripts.
+func (dcr *ExchangeWallet) blockMaybeContainsScripts(blockHash string, scripts [][]byte) (bool, error) {
```

Comment:
Let's remember to refactor this method in terms of the `blockFilter` struct and the `getBlockFilterV2` methods slated to be added in https://github.com/decred/dcrdex/pull/788

Reply:
Yeah, this will likely go in favor of that. Wanted a simple version in first.